.editorconfig

```diff
@@ -63,35 +78,30 @@ csharp_space_between_method_declaration_name_and_open_parenthesis = true
 csharp_space_between_method_call_name_and_opening_parenthesis = true
 csharp_space_after_keywords_in_control_flow_statements = true
 
-# Newline settings
-csharp_new_line_before_open_brace = accessors,types,methods,properties,events,indexers
-csharp_new_line_before_else = false
-csharp_new_line_before_catch = false
-csharp_new_line_before_finally = false
-csharp_new_line_before_members_in_object_initializers = true
-csharp_new_line_before_members_in_anonymous_types = true
-
 # Switch indentation
 csharp_indent_switch_labels = false
 
 # Prefer "var" everywhere it's apparent
 csharp_style_var_for_built_in_types = true:suggestion
 csharp_style_var_when_type_is_apparent = true:suggestion
-csharp_style_var_elsewhere = false:suggestion
-
-# Prefer method-like constructs to have a block body
-csharp_style_expression_bodied_methods = false:none
-csharp_style_expression_bodied_constructors = false:none
-csharp_style_expression_bodied_operators = false:none
+csharp_style_var_elsewhere = true:suggestion
 
 # Prefer property-like constructs to have an expression-body
-csharp_style_expression_bodied_properties = false:none
-csharp_style_expression_bodied_indexers = false:none
-csharp_style_expression_bodied_accessors = false:none
+csharp_style_expression_bodied_properties = true:suggestion
+csharp_style_expression_bodied_indexers = true:suggestion
+csharp_style_expression_bodied_accessors = true:suggestion
 
 # Suggest more modern language features when available
 csharp_style_pattern_matching_over_is_with_cast_check = true:suggestion
 csharp_style_pattern_matching_over_as_with_null_check = true:suggestion
 csharp_style_inlined_variable_declaration = true:suggestion
-csharp_style_throw_expression = false:suggestion
+csharp_style_throw_expression = true:suggestion
 csharp_style_conditional_delegate_call = true:suggestion
+
+# Newline settings
+csharp_new_line_before_open_brace = types,methods
```

Comment:
@chkn do you mind this applying to all of these `accessors,types,methods,properties,events,indexers` rather than just types and methods ?

Reply:
I think that adds a lot of extra white space. It gets especially obnoxious with things like properties. I think the Mono coding convention is just `methods` here. Personally, I’d be more inclined to remove `types` to align 100% with mono than going the other direction by adding things

Reply:
I love white space. But it isn't a deal breaker for me.